release_notes/2.1.7.md

```diff
@@ -0,0 +1 @@
+* Bug fixes to logout of RSA Archer API at end of actions and return success for 0 records of 'list tickets' action
```

Comment:
This file can be removed as the pipeline automatically creates a version-specific md file for release notes from the unreleased.md file on merge.

Reply:
Done